### ISSUE TITLE
Support for streaming for bash commands

### ISSUE THREAD

**Issue by rbren** [OpenHands/software-agent-sdk#1765]
Currently you have to wait for a bash command to complete or timeout to see any of the output. It'd be nice to get the progressive output via a stream.

This will be hard, we'll probably need a second websocket or sth

### If you find this feature request or enhancement useful, make sure to add a 👍 to the issue

**Comment by diwu-sf** [comment]
This should be easier with tmux, and can just take a tmux screen output and poll on it periodically

**Comment by anemo-hypostasis** [comment]
Any progress on this? Looking forward to this feature.

**Comment by jatinkrmalik** [comment]
Also, I think it would be really cool to have an option to **auto expand all bash commands** by default or as an opt in.

I am very curious as an engineer to see what commands the agent is executing.

Exhibit A: 

![Image](https://github.com/user-attachments/assets/892bf731-31fb-4691-8296-d90192714248)

**Comment by jpelletier1** [comment]
@rbren @xingyaoww Is this something we are still working on?

**Comment by jpelletier1** [comment]
@rbren How are you thinking about this in context of V1? Still a valid roadmap request?

**Comment by jpelletier1** [comment]
@xingyaoww Curious if you have any thoughts/feedback on priority for this?

**Comment by xingyaoww** [comment]
We probably need to move this issue to SDK repo, but on the other hand, i don't think this issue is very high priority any more (i barely looked at agent's action observation nowadays 😓 )

**Comment by jpelletier1** [comment]
@xingyaoww Agreed... going to add the "stale" label for now and see if we hear any additional feedback prompting to remove it :)

**Comment by enyst** [comment]
🤔 Actually, I think you're right, maybe we can close this issue. If people care about it, we could always look at new requests... Doubtful, we've moved on from looking at every event 😅
